#goal definition
mav_trajectory_generation_ros/PVAJS_array flatStates
---
#result definition
bool success
---
#feedback
mav_trajectory_generation_ros/PVAJS currentFlatState